scripts/build/ci-deploy/Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM debian:testing-20210111-slim
+FROM debian:bullseye-slim
 
 # Use ARG so as not to persist environment variable in image
-ARG GOVERSION=1.17.8 \
-  GO_CHECKSUM=980e65a863377e69fd9b67df9d8395fd8e93858e7a24c9f55803421e453f4f99 \
-  DEBIAN_FRONTEND=noninteractive
```

Comment:
Why do you need to upgrade Golang? This would amount to a change across the whole of Grafana. Again, something that would require a separate PR.

Reply:
I saw that golang used here only to build aptly and checked that it works with last version of golang.

I will create more conservative changes for now (but still suggest to have and check updated aptly, as it should support gpg2 in master, what do you think?)